src/Mainpage/modules/typescript/summary.ts

```
@@ -0,0 +1,55 @@
+		const pageKeys = Object.keys(res.query.pages);
+
+		const summaryUrlHtml = `<a href="https://youshou.wiki/wiki/${titleStr}">更多消息，请查阅本期双周报详细信息</a>`;
+		const {pages} = res.query;
+		const [pageKey] = pageKeys;
+
+		const page = pageKey ? pages[pageKey] : undefined;
+		let extract: string;
+		if (page) {
+			extract =
+				page && page.extract
+					? page.extract.replace('更多消息，请查阅本期双周报详细信息', summaryUrlHtml).replace(/…$/g, '')
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)